server/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const path = require('path');
+
+const app = express();
+
+app.use(express.static(path.join(__dirname, '../public')));
+
+app.get('/test', (req, res) => {
+  res.send('It works!');
+});
+
+app.listen(3020);
```

Comment:
It is typically convention to define static numbers as a const to reduce confusion surrounding what a static number represents in a given context. Here I am making the assumption that that is your port number, but it could be more clear.

Reply:
Sounds good.